src/validateInvocation.ts

```diff
@@ -21,6 +21,15 @@ export default async function validateInvocation(
     );
   }
 
+  if (config.disableSslVerification) {
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+    context.logger.publishEvent({
+      name: 'disable_ssl_certs',
+      description:
+        'Disabling SSL Certs. NOT RECOMMENDED: Please install TLS certificates from https://letsencrypt.org/',
```

Comment:
🔥 

I think we might confuse some folks who think (for some reason) we are saying Let's Encrypt is the only option. Also, making reference to SSL consistent because TLS is the correct terminology (technically, SSL is dead; long live TLS).
```suggestion
        'Disabling TLS certificate verification. NOT RECOMMENDED: Please install valid TLS certificates into Rapid7 server.',
```